src/components/FlowRunTimeline.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="flow-run-timeline-container">
+    <span class="flow-run-timeline-container__beta-badge">
+      Beta
+    </span>
+    <FlowRunTimeline
+      v-if="graphData.length > 0"
+      :graph-data="graphData"
+      :is-running="isRunning"
+    />
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { FlowRunTimeline, TimelineNodeData } from '@prefecthq/graphs'
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed } from 'vue'
+  import { useWorkspaceApi } from '@/compositions'
+  import { FlowRun, GraphNode } from '@/models'
+
+  const props = defineProps<{
+    flowRun: FlowRun,
+  }>()
+
+  const api = useWorkspaceApi()
+
+  const graphSubscription = useSubscription(api.flowRuns.getFlowRunsGraph, [props.flowRun.id], { interval: 1000 })
```

Comment:
```suggestion
  const graphSubscription = useSubscription(api.flowRuns.getFlowRunsGraph, [props.flowRun.id], { interval: 5000 })
```

Polling for this every second is extremely aggressive; I'd recommend you bump this up to at least 5 seconds.